pkg/util/utils.go

```diff
@@ -79,18 +80,20 @@ func ParseChanges(option string) (key string, vals []string, err error) {
 	// 3. key ["value","value1"]
 	if strings.Contains(option, " ") {
 		// This handles 2 & 3 conditions.
-		var val string
-		tokens := strings.SplitAfterN(option, " ", 2)
+		tokens := strings.SplitN(option, " ", 2)
 		if len(tokens) < 2 {
 			return "", []string{}, fmt.Errorf("invalid key value %s", option)
 		}
 		key = strings.Trim(tokens[0], " ") // Need to trim whitespace part of delimiter.
-		val = tokens[1]
-		if strings.Contains(tokens[1], "[") && strings.Contains(tokens[1], "]") {
-			//Trim '[',']' if exist.
-			val = strings.TrimLeft(strings.TrimRight(tokens[1], "]"), "[")
+
+		if strings.Contains(tokens[1], "[") || strings.Contains(tokens[1], "]") {
```

Comment:
probably doesn't matter, but should this be an "and" rather than an "or"?

Reply:
An or - so we can detect invalid JSON arrays like `["hello", "world"` (missing `]`) and similar